### ISSUE TITLE
video capture window is not working 

### ISSUE THREAD

**Issue by krshashank963** [trygvea/face-recognition-dlib#1]
I tried this code in latest version of spyder everything was good untill video capture window open.
  i tried other testing code for camera permission it worked but with this code only a blank window opened

**Comment by trygvea** [comment]
Do you get any error messages?

**Comment by krshashank963** [comment]
yes this is error message and after thet camera get on means its led is on 

    flags = cv2.CASCADE_SCALE_IMAGE)

error: OpenCV(4.2.0) C:\projects\opencv-python\opencv\modules\objdetect\src\cascadedetect.cpp:1689: error: (-215:Assertion failed) !empty() in function 'cv::CascadeClassifier::detectMultiScale'

I put all the cv lib in perfect directory
